server_development/topics/user-storage/simple-example.adoc

```diff
@@ -257,7 +257,7 @@ The class files for our provider implementation should be placed in a jar.  You
 org.keycloak.examples.federation.properties.FilePropertiesStorageFactory
 ----
 
-Once you create the jar you can deploy it using regular {appserver_name} means: copy the jar into the `standalone/deployments/` directory or using the JBoss CLI.
+Once you create the jar you can deploy it using regular {appserver_name} means: copy the jar into the `deploy/` directory or using the JBoss CLI.
```

Comment:
Not sure about this change? There is no "deploy/" directory inside Keycloak/RHSSO. I think the correct dir should be really "standalone/deployments/" . Did you added this change by accident?

Reply:
This is also a change that I see in my fork of the master. I didn't make this change.  It appears to already be published: https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html-single/server_developer_guide/index#simple-example.adoc
However, if someone made this change by mistake, I can fix it. What would you recommend?

Reply:
Similar case like above. This was incorrect in old keycloak and RHSSO 7.3 documentation, then one user from the community fixed it in this commit https://github.com/keycloak/keycloak-documentation/commit/5519fe7bc09779a2e47a5ab4e436e8f2dee5670f and then in this PR, you're reverting the fix back, which is not correct.